astropy/nddata/mixins/tests/test_ndio.py

```diff
@@ -12,5 +20,415 @@ class NDDataIO(NDIOMixin, NDData):
 
 def test_simple_write_read(tmpdir):
     ndd = NDDataIO([1, 2, 3])
-    assert hasattr(ndd, 'read')
-    assert hasattr(ndd, 'write')
+    ndd.read
```

Comment:
Actually I wanted to delete the test altogether because if it doesn't have read/write all the subsequent tests will fail, forgot to change it back after I tried my first test there and found out it's easier to do the tests as a class.